samples/app-hello-world/lib/utils.py

```diff
@@ -0,0 +1,101 @@
+import urllib3
+import os
+import json
+import base64
+from urllib.parse import urlencode
+import time
+
+
+arbiter = os.environ['DATABOX_ARBITER_ENDPOINT']
+hostname = os.environ['DATABOX_LOCAL_NAME']
+
+arbiterToken = open("/run/secrets/ARBITER_TOKEN", "rb").read()
+arbiterTokenbase64 = base64.b64encode(arbiterToken)
+
+CM_HTTPS_CA_ROOT_CERT = open("/run/secrets/DATABOX_ROOT_CA").read()
+
+if CM_HTTPS_CA_ROOT_CERT is not None:
+    http = urllib3.PoolManager(
+    ca_certs='/run/secrets/DATABOX_ROOT_CA')
+else:
+    print('Warning: No HTTPS root certficate provided so Databox HTTPS certificates will not be checked')
+
```

Comment:
fine for now, but we probably want to look at this again. not checking certs is often a worse default than just giving up and failing